sdk/azcore/README.md

```diff
@@ -0,0 +1,39 @@
+# Azure Core Client Library for Go
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/azcore)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore)
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/go/go%20-%20azcore%20-%20ci?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1843&branchName=master)
+[![Code Coverage](https://img.shields.io/azure-devops/coverage/azure-sdk/public/1843/master)](https://img.shields.io/azure-devops/coverage/azure-sdk/public/1843/master)
```

Comment:
Just curious, how do you get this image?

Reply:
I searched for "azure devops code coverage badge" and found https://shields.io/category/coverage

Reply:
I see, thanks for the info! :)